help.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+if [[ -n $(sudo docker ps -qf "name=leetcode") ]]
+then 
+	:
+elif [[ -n $(sudo docker ps -qaf "name=leetcode") ]]
+then
+	sudo docker start leetcode
```

Comment:
The script needs the name `leetcode:v1` and `leetcode` multiple times and sometimes they are confusing. Make variables for them.